src/graph/entities/qg_edge.h

```diff
@@ -62,6 +62,12 @@ bool QGEdge_VariableLength
 	const QGEdge *e
 );
 
+// determine whether this is a edge with a range of length defined (variable length [*minHops..maxHops] or fixed length [*Hops])
```

Comment:
I think comment need update here

Reply:
When I was updating the comment I noticed that maybe this function should be replaced by a ```QGEdge_LengthOne()```. See comment below.